### PR TITLE
Add new permissions to aws doc and include postgres in rds enhanced integration section

### PR DESCRIPTION
**Pull request by DanielLanger**

### CODE REVIEW

**Comment by garnermccloud** [comment]
👍

File: content/integrations/aws.md [hunk 1]
```diff
@@ -105,7 +106,10 @@ Note: The GovCloud and China regions do not currently support IAM role delegatio
                 "sns:List*",
                 "sns:Publish",
                 "sqs:ListQueues",
-                "support:*"
+                "support:*",
```
Comment (irabinovitch) [inline content/integrations/aws.md]: This seems pretty wide? Can we tighten a bit or do we really need it all?

Comment (DanielLanger) [inline content/integrations/aws.md]: http://docs.aws.amazon.com/IAM/latest/UserGuide/list_trustedadvisor.html unfortunately it requires *

Comment (DanielLanger) [inline content/integrations/aws.md]: @irabinovitch

**Pull request merged**